crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -699,12 +764,15 @@ fn build_lattice_methods_by_wit_interface(
             // across the lattice, in a <CamelCaseModule><CamelCaseInterface><CamelCaseFunctionName> pattern
             // (ex. MessagingConsumerRequestMultiInvocation)
             let lattice_method_name = LitStr::new(
-                format!(
-                    "{}{}/Message.{}",
-                    wit_ns,
-                    wit_package,
-                    f.sig.ident.to_string().to_upper_camel_case()
-                )
+                match (wit_ns, wit_package) {
+                    ("wasmcloud", "keyvalue") => {
+                        format!("KeyValue.{}", f.sig.ident.to_string().to_upper_camel_case())
+                    }
+                    (ns, pkg) => format!(
+                        "{ns}:{pkg}/Message.{}",
```

Comment:
Could you help me understand why `Message` is hardcoded and not generated from the interface name?

Reply:
`Message` was in [earlier prototypes](https://github.com/thomastaylor312/nats-messaging-wit/blob/main/src/messaging.rs#L40) on which this code was built & tested, which is why it is [in the repo right now](https://github.com/wasmCloud/wasmCloud/blob/main/crates/provider-wit-bindgen/src/lib.rs#L703). 

Note that Github may show certain code that is changed as additions rather than same-line modifications -- this code isn't new. 

If expanded options (see other comment below) lands in this PR, then this area will be different